CryptoPrices/Sources/Domain/Sources/UseCaseProtocol/CoinDetailUseCaseProtocol.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  CoinDetailUseCaseProtocol.swift
+//
+//  Created by Khanh on 27/12/2022.
+//
+
+import Entities
+
+// sourcery: AutoMockable
+public protocol CoinDetailUseCaseProtocol {
+
+    func execute(id: String) async throws -> CoinDetail
+}
```

Comment:
Add `// sourcery: AutoMockable` here is correct because it has a method call that we want to tamper its implementation and returning types for unit testing 👍